src/App.js

```diff
@@ -3,26 +3,52 @@ import Movies from './Movies';
 import Movie from './Movie';
 import Nav from './Nav';
 import movieData from './data';
+import fetchCalls from './apiCalls'
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      movies:[],
+      singleMovieId:'',
       singleMovie:'',
```

Comment:
What is the purpose of storing both the single movie ID _and_ the whole single movie (or at least I think you're eventually storing the whole single movie object)? Can you get by with storing only one of these?